### PR TITLE
Add Windows support for Android Tools

### PR DESCRIPTION
**Pull request by goderbauer**
Required to run tests on Windows bots.

### CODE REVIEW

File: dev/bots/README.md [hunk 1]
```diff
@@ -87,3 +87,46 @@ Support for [cross-repository
 recipes](https://github.com/luci/recipes-py/blob/master/doc/cross_repo.md) is
 in-progress.  If you view the git log of this directory, you'll see we initially
 tried, but it's not quite ready.
+
+# Android Tools
+
+Instructions to update the Andoird Tools version that the bots download by executing `download_andoird_tools.py`.
+
+## How to update Android SDK on GCS
```
Comment (Hixie) [inline dev/bots/README.md]: GCS?

Comment (Hixie) [inline dev/bots/README.md]: Oh Google Cloud Storage. We should expand the acronym here and in the heading below.

File: dev/bots/README.md [hunk 2]
```diff
@@ -87,3 +87,46 @@ Support for [cross-repository
 recipes](https://github.com/luci/recipes-py/blob/master/doc/cross_repo.md) is
 in-progress.  If you view the git log of this directory, you'll see we initially
 tried, but it's not quite ready.
+
+# Android Tools
+
+Instructions to update the Andoird Tools version that the bots download by executing `download_andoird_tools.py`.
```
Comment (Hixie) [inline dev/bots/README.md]: typo in filename here

**Comment by Hixie** [comment]
Why Python instead of Dart? :-)

**Comment by Hixie** [comment]
RSLGTM

**Comment by goderbauer** [comment]
Python because the original script in the now-defunct mojo repo [1] for this was also python.

[1] https://github.com/domokit/mojo/tree/master/tools/android

**Pull request merged**